infoschema/tables_test.go

```diff
@@ -867,6 +889,34 @@ func (s *testClusterTableSuite) TestSelectClusterTable(c *C) {
 	}
 }
 
+func (s *testClusterTableSuite) TestClusterTablePrivileges(c *C) {
+	tk := s.newTestKitWithRoot(c)
+	// More tests about the Cluster_TiDB_Trx's privileges.
+	tk.MustExec("create user 'testuser'@'localhost'")
+	tk.MustExec("create user 'testuser2'@'localhost'")
+	tk.MustExec("grant process on *.* to 'testuser2'@'localhost'")
+	tk.MustExec("flush privileges")
```

Comment:
Is this needed? I think it only should be needed when manipulating the mysql.* tables directly, not when using grant statements.

Reply:
I see, got it